shape_msgs/SolidPrimitive box
geometry_msgs/Point center_point
